graphics/zimg/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        sekrit-twc zimg 2.9.3 release-
+
+categories          graphics
+platforms           darwin
+license             WTFPL
+
+description         Scaling, colorspace conversion, and dithering library
+
+long_description    The \"z\" library implements the commonly required image \
+                    processing basics of scaling, colorspace conversion, and \
+                    depth conversion. A simple API enables conversion between \
+                    any supported formats to operate with minimal knowledge \
+                    from the programmer. All library routines were designed \
+                    from the ground-up with correctness, flexibility, and \
+                    thread-safety as first priorities. Allocation, buffering, \
+                    and I/O are cleanly separated from processing, allowing \
+                    the programmer to adapt \"z\" to many scenarios.
+
+checksums           ${distname}${extract.suffix} \
+                    rmd160  13991643226cf16e13ecfa2cbc2131162a50fb8a \
+                    sha256  d4e5c9c61563236eb768d77caa53620b456991168e7670bb4d2aa4995b6a9560 \
+                    size    266250
+
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+compiler.thread_local_storage   yes
```

Comment:
This is still failing to build with:
```
error: thread-local storage is not supported for the current target
```
So this must be using the C++11 flavor of thread-local storage, so the port also needs:
```
compiler.cxx_standard 2011
```

Reply:
Added